__tests__/main.test.ts

```diff
@@ -87,6 +97,25 @@ describe("action", () => {
         } as Inputs,
         "input provided for api_url not in expected format",
       ],
+      [
+        "identity_url",
+        {
+          accessToken: TEST_ACCESS_TOKEN,
+          secrets: TEST_SECRETS,
+          identityUrl: INVALID_URL,
+          apiUrl: DEFAULT_API_URL,
+        } as Inputs,
+        "input provided for identity_url not in expected format",
+      ],
+      [
+        "cloud_region",
+        {
+          accessToken: TEST_ACCESS_TOKEN,
+          secrets: TEST_SECRETS,
+          cloudRegion: INVALID_REGION,
```

Comment:
The `cloudRegion` have to be specified in the `OptionalInputs` type and used in `mockInputs` for this test to work (lines 370+)